Dev/Install/config-form.html

```diff
@@ -6,7 +6,7 @@
 	<head>
 		<title>SilverStripe CMS / Framework Installation</title>
 		<meta http-equiv="Content-type" content="text/html; charset=utf-8">
-		<script type="application/javascript" src="<?php echo FRAMEWORK_NAME; ?>/thirdparty/jquery/jquery.js"></script>
+		<script type="application/javascript" src="<?php echo FRAMEWORK_NAME; ?>/admin/thirdparty/jquery/jquery.js"></script>
```

Comment:
Done, using a CDN loses the ability to use the Installer properly if the user goes offline between downloading SilverStripe and using the installer, which I would imagine isn't common,